packages/logger/lib/logger.js

```diff
@@ -1,6 +1,6 @@
 const pino = require('pino').default;
 const serializeError = require('@dotcom-reliability-kit/serialize-error');
-const { default: clone } = require('@ungap/structured-clone');
+const clone = require('lodash.clonedeep');
```

Comment:
Sorry for being annoying, but I'd require `cloneDeep`. It seems like lodash has a separate method called `clone`

```suggestion
const cloneDeep = require('lodash.clonedeep');
```

Reply:
I think I'm OK with it staying as-is, as it results in an easier-to-read diff and the library we use to clone is kind of an implementation detail. We don't install lodash.clone so don't think that'd cause confusion, and we also didn't use the name `structuredClone` when we were using that library